### PR TITLE
DOCSP-43402: link to migration guide + clarify lang

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-43402>
Staging - https://deploy-preview-381--docs-golang.netlify.app/whats-new/#what-s-new-in-2.0

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/whats-new.txt [hunk 1]
```diff
@@ -51,12 +51,16 @@ Learn what's new in:
 What's New in 2.0
 -----------------
 
-The 2.0 {+driver-short+} release supports various new features, provides
-a smoother timeout experience, and implements clearer, more idiomatic Go
+The 2.0 {+driver-short+} release supports OpenID Connect (OIDC)
```
Comment (jtazin) [inline source/whats-new.txt:54]: This looks good in general, but let me just figure out if we're ready to advertise OIDC

File: source/whats-new.txt [hunk 2]
```diff
@@ -51,12 +51,16 @@ Learn what's new in:
...
+authentication, provides a smoother Client-Side Operations Timeout
```
Comment (mongoKart) [inline source/whats-new.txt:55]: S: we don't capitalize this elsewhere, though I'm not sure which is correct. See, e.g., https://www.mongodb.com/docs/languages/python/pymongo-driver/current/connect/csot/

```suggestion
authentication, provides a smoother client-side operations timeout
```

File: source/whats-new.txt [hunk 3]
```diff
@@ -99,6 +103,10 @@ In this section, you can learn more about these changes.
 
 This release includes the following improvements and fixes:
 
+- Support for OpenID Connect (OIDC) authentication. To learn more, see
+  :atlas:`Authentication and Authorization with OIDC/OAuth 2.0
```
Comment (mongoKart) [inline source/whats-new.txt:107]: Q: will this be a link to the Go OIDC section when that's done?

**Review by mongoKart — APPROVED** [review]
lgtm w/ comments

**Pull request merged**